src/plugins/intel_cpu/src/emitters/snippets/cpu_runtime_configurator.cpp

```diff
@@ -28,7 +28,9 @@
 #    include "transformations/snippets/x64/pass/lowered/external_repacking_adjuster.hpp"
 #endif
 #ifdef OPENVINO_ARCH_ARM64
-#    include "transformations/snippets/aarch64/pass/lowered/gemm_copy_b_loop_ports_adjuster.hpp"
+#    ifdef OV_CPU_WITH_KLEIDIAI
```

Comment:
What's about to combine these conditions?
```
#if defined (OPENVINO_ARCH_ARM64) && defined(OV_CPU_WITH_KLEIDIAI)
```

The same idea is for the similar lines below (and for any other similar places if they exist)